### PR TITLE
Fix the setting of the content-type for Oneoffixx templates

### PR DESCRIPTION
**Pull request by Rotonen**
Seems I actually built all the machinery we needed, but then forgot to to actually plug it in in the end.

Original PR: https://github.com/4teamwork/opengever.core/pull/4282

🐷 will get fed.

### CODE REVIEW

**Comment by Rotonen** [comment]
And we ended up testing this functionality against the shadow document in the fixture, which staticly sets it in the document annotations.

Now testing the annotations of an actually created shadow document.

File: opengever/oneoffixx/tests/test_oneoffixx.py [hunk 1]
```diff
@@ -128,9 +129,29 @@ def test_document_creation_from_oneoffixx_template_creates_shadow_doc(self, brow
...
+            'application/vnd.openxmlformats-officedocument.wordprocessingml'
+            '.document',
+            annotations['content-type'],
+        )
+
+        whitelisted_content_types = [
+            template['content-type']
+            for template in whitelisted_template_types.values()
+        ]
+        self.assertIn(annotations['content-type'], whitelisted_content_types)
```
Comment (njohner) [inline opengever/oneoffixx/tests/test_oneoffixx.py]: This seems a bit disconnected from the rest of the test, and the added value seems slim.

**Comment by njohner** [comment]
Oh forgot, but can you reference the original PR in the description?

File: opengever/oneoffixx/tests/test_oneoffixx.py (hunk 1 above)
Comment (Rotonen) [reply opengever/oneoffixx/tests/test_oneoffixx.py]: The annotations having been set properly are an integral part of having created a shadow document from an Oneoffixx template. Where else should the assertions go?

**Comment by Rotonen** [comment]
@njohner amended the PR body.

File: opengever/oneoffixx/tests/test_oneoffixx.py (hunk 1 above)
Comment (njohner) [reply opengever/oneoffixx/tests/test_oneoffixx.py]: Sorry maybe that wasn't clear. Testing the annotations above is fine with me. I was just wondering about that last assertion here, where you test that the `annotations['content-type']` is in the `whitelisted_content_types`. You have already tested what `annotations['content-type']` was, just above, so here you are affectively testing that `'application/vnd.openxmlformats-officedocument.wordprocessingml.document'` is in `whitelisted_content_types`. I was wondering whether that made much sense or not.

Comment (Rotonen) [reply opengever/oneoffixx/tests/test_oneoffixx.py]: As the content type is pulled from that whitelist based on the meta template ID, I felt it prudent to add that assertion as well.

Comment (njohner) [reply opengever/oneoffixx/tests/test_oneoffixx.py]: I approved the PR because I did not have a strong feeling about this, so feel free to leave it in.

**Pull request merged**